lnrpc/rpc.proto

```diff
@@ -2647,6 +2654,26 @@ message PayReq {
     map<uint32, Feature> features = 13 [json_name = "features"];
 }
 
+enum FeatureBit {
+        DATALOSS_PROTECT_REQ = 0;
+        DATALOSS_PROTECT_OPT = 1;
+        INITIAL_ROUING_SYNC = 3;
+        UPFRONT_SHUTDOWN_SCRIPT_REQ = 4;
+        UPFRONT_SHUTDOWN_SCRIPT_OPT = 5;
+        GOSSIP_QUERIES_REQ = 6;
+        GOSSIP_QUERIES_OPT = 7;
+        TLV_ONION_REQ = 8;
+        TLV_ONION_OPT = 9;
+        EXT_GOSSIP_QUERIES_REQ = 10;
+        EXT_GOSSIP_QUERIES_OPT = 11;
+        STATIC_REMOTE_KEY_REQ = 12;
+        STATIC_REMOTE_KEY_OPT = 13;
+        PAYMENT_ADDR_REQ = 14;
+        PAYMENT_ADDR_OPT = 15;
+        MPP_REQ = 16;
+        MPP_OPT = 17;
+}
```

Comment:
We also add the non-payment related ones for if people query node info for example?